src/components/InstrumentsSidebar.tsx

```diff
@@ -0,0 +1,54 @@
+import { Tooltip, TooltipContent, TooltipProvider, TooltipTrigger } from '@/components/ui/tooltip'
+import { cn } from '@/lib/utils'
+import { ReactNode, useState } from 'react'
+
+type ButtonData = {
+  tooltip: string
+  icon: ReactNode
+}
+
+interface InstrumentsSidebarProps {
+  buttonsData: ButtonData[]
+}
+
+const InstrumentsSidebar = ({ buttonsData }: InstrumentsSidebarProps) => {
+  const [activeButtonIndex, setActiveButtonIndex] = useState(0)
+
+  const buttonClickHandler = (button: ButtonData, buttonIndex: number) => {
+    setActiveButtonIndex(buttonIndex)
+    console.log(button.tooltip)
```

Comment:
I suppose this was added for debug. Let's not add console logs to develop branch